python/pyspark/sql/connect/client/reattach.py

```diff
@@ -335,6 +321,5 @@ def __del__(self) -> None:
 
 class RetryException(Exception):
     """
-    An exception that can be thrown upstream when inside retry and which will be retryable
-    regardless of policy.
+    An exception that can be thrown upstream when inside retry and which is always retryable
     """
```

Comment:
nit: maybe move to retries.py? I think it shouldn't have landed in this file in the first place.

Reply:
Moved